projects/swimlane/ngx-graph/src/lib/graph/graph.component.ts

```diff
@@ -489,11 +503,7 @@ export class GraphComponent extends BaseChartComponent implements OnInit, OnChan
       });
     }
 
-    // Calculate the height/width total, but only if we have any nodes
-    if (this.graph.nodes && this.graph.nodes.length) {
```

Comment:
This code should be removed?

Reply:
Does the minimap code sets the height of entire graph in updateGraphDims call?

Reply:
yes, all of this logic is in the updateMinimap method.

Also, I optimised the calculation of `graphDims` (should be faster with larger amount of nodes since I'm only doing a single pass now), and it is more precise (accounting for negative values in the position if you drag the node to the left or top).